smsdk/client_v0.py

```diff
@@ -317,12 +317,22 @@ def inner(
                 only_names = schema["name"].tolist()[:50]
                 kwargs["_only"] = only_names
             else:
-                if ("End Time" not in kwargs["_only"]) and (
-                    "endtime" not in kwargs["_only"]
+                if (
+                    ("End Time" not in kwargs["_only"])
+                    and ("endtime" not in kwargs["_only"])
+                    and ("Cycle End Time" not in kwargs["_only"])
                 ):
```

Comment:
Not sure if this is more efficient
```suggestion
                if all(
                  i not in {"End Time", "endtime", "Cycle End Time"}
                  for i in kwargs["_only"]
                ):
```